microbootstrap/bootstrappers/fastapi.py

```diff
@@ -57,15 +56,15 @@ def bootstrap_before(self) -> dict[str, typing.Any]:
             "version": self.instrument_config.service_version,
         }
 
-    def bootstrap_after(self, application: fastapi.FastAPI) -> fastapi.FastAPI:
+    def bootstrap_after(self, application: fastapi.FastAPI) -> fastapi.FastAPI:  # type: ignore[override]
```

Comment:
I think it would be possible to remove type-ignore comments if we configure `ApplicationT` type variable

Reply:
solved